Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidDevice.cs

```diff
@@ -0,0 +1,62 @@
+#if UNITY_EDITOR || UNITY_ANDROID
+
+using System;
+using UnityEngine.InputSystem.Controls;
+using UnityEngine.InputSystem.Layouts;
+
+namespace UnityEngine.InputSystem.Android.LowLevel
+{
+    // <summary>
+    /// Input device that represents an Android handheld device.
+    /// </summary>
+    [InputControlLayout(displayName = "Android Device", hideInUI = true)]
+    public class AndroidDevice : InputDevice
+    {
+        [InputControl(synthetic = true)]
+        public ButtonControl button { get; protected set; }
+        
+        /// <inheritdoc />
+        protected override void OnAdded()
+        {
+            base.OnAdded();
+            InputSystem.onSettingsChange += OnSettingsChange;
+        }
+
+        /// <inheritdoc />
+        protected override void OnRemoved()
+        {
+            InputSystem.onSettingsChange -= OnSettingsChange;
+            base.OnRemoved();
+        }
+        
+        private void OnSettingsChange()
+        {
+            backButtonLeavesApp = InputSystem.settings.android.backButtonLeavesApp;
+        }
+        
+        private bool backButtonLeavesApp
+        {
+            get 
+            {
+                var command = GetCustomCommand.Create();
```

Comment:
For how far back is this command supported in native run-time? I assume the command would fail with an error if not supported in backend and this defaults to false, correct?